scripts/install.sh

```diff
@@ -140,8 +141,14 @@ else
 	HOOK_FILE="${PATH_GIT_LOCAL}${HOOK_DIR}${HOOK_NAME}"
 fi
 
-printf -- " - Downloading git hook...\n"
-curl -s "$GITHUB_SCRIPT_URL" > "$HOOK_FILE"
+if [ "$should_use_file_from_github_url" = true ] ; then
```

Comment:
Also a very nice feature 🙌 I might also like to add this to the install script

Reply:
Thanks :) I was too lazy to add it to the advanced option and I used it only for testing